srcs/layouts/latn_qwerty_tly.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<keyboard name="QWERTY (Talysh New Latin)" script="latin">
+  <row>
+    <key key0="g" key2="1" key4="q"/>
+    <key key0="ú" key2="2" key4="ü"/>
+    <key key0="e" key2="3" key3="\#"/>
+    <key key0="ê" key2="4" key3="!"/>
+    <key key0="r" key2="5" key3="$"/>
+    <key key0="t" key2="6" key3="%"/>
+    <key key0="y" key2="7" key3="^"/>
+    <key key0="u" key2="8" key3="û"/>
+    <key key0="i" key2="9" key3="*"/>
+    <key key0="o" key1="ö" key2="0" key3="("/>
+    <key key0="p" key3=")"/>
+    <key key0="q" key3="&amp;" key4="ğ"/>
+  </row>
+  <row>
+    <key shift="0.5" key0="á" key2="`" key4="a"/>
```

Comment:
You should drop this `shift="0.5"` attribute:

![shot](https://github.com/Julow/Unexpected-Keyboard/assets/2310568/21bf46c7-2df2-4e87-b64e-27f256e48d02)

(some symbols are added by my own configuration)